coral/src/app/features/configuration/users/components/UsersTable.tsx

```diff
@@ -0,0 +1,119 @@
+import { Box, DataTable, DataTableColumn, EmptyState } from "@aivenio/aquarium";
+import { User } from "src/domain/user";
+import uniqueId from "lodash/uniqueId";
+
+type UsersProps = {
+  users: User[];
+  ariaLabel: string;
+};
+
+interface UsersRow {
+  id: string;
+  username: User["username"];
+  fullname: User["fullname"];
+  role: User["role"];
+  team: User["team"];
+  mailid: User["mailid"];
+  switchTeams: User["switchTeams"];
+  switchAllowedTeamNames: User["switchAllowedTeamNames"];
+}
+
+const UsersTable = ({ users, ariaLabel }: UsersProps) => {
+  const columns: Array<DataTableColumn<UsersRow>> = [
+    {
+      type: "text",
+      field: "username",
+      headerName: "Username",
+    },
+    {
+      type: "text",
+      field: "fullname",
+      headerName: "Name",
+    },
+    {
+      type: "status",
+      field: "role",
```

Comment:
I think `field` is only necessary using the `'text`. I get an TS error in my editor for this property :o

```
Type '{ type: "status"; field: string; headerName: string; status: ({ role }: UsersRow) => { status: "info" | "neutral"; text: string; }; }' is not assignable to type 'DataTableColumn<UsersRow>'.
  Object literal may only specify known properties, and 'field' does not exist in type 'Column & { width?: number | `${number}%` | undefined; } & ColumnBase<UsersRow> & CustomSortable<UsersRow> & { type: "status"; status: (row: UsersRow, index: number, rows: UsersRow[]) => TableStatus | undefined; }'.
```

Reply:
Ah, will update!

Reply:
https://github.com/Aiven-Open/klaw/pull/1971/commits/5afad6eeb25a4be8a1756d0538de7a7a61291b18